src/Akeneo/Pim/Enrichment/Bundle/Command/MigrateToUuid/MigrateToUuidCommand.php

```diff
@@ -17,6 +19,9 @@
 class MigrateToUuidCommand extends Command
 {
     protected static $defaultName = 'pim:product:migrate-to-uuid';
+    private static $DQI_JOB_NAME = 'data_quality_insights_evaluations';
+    private static $WAIT_TIME = 30;
```

Comment:
```suggestion
    private static $WAIT_TIME_IN_SECONDS = 30;
```

Otherwise we need to search in the code the unit